GDJS/Runtime/gd.js

```diff
@@ -135,7 +135,7 @@ gdjs.toDegrees = function (angleInRadians) {
  * of the type of the object is "TextObject::Text".
```

Comment:
These changes are not right, the parameters are NOT `gdjs.Runtimeobject`, which would expect an instance of the class, but the class itself.
If it's causing problems, try `Class<gdjs.RuntimeObject>`?